src/locale/i18n.ts

```diff
@@ -78,6 +79,14 @@ export async function dynamicActivate(locale: AppLanguage) {
       ])
       break
     }
+    case AppLanguage.el: {
+      i18n.loadAndActivate({locale, messages: messagesEl})
+      await Promise.all([
+        import('@formatjs/intl-pluralrules/locale-data/el'),
```

Comment:
```suggestion
        import('@formatjs/intl-datetimeformat/locale-data/el'),
        import('@formatjs/intl-pluralrules/locale-data/el'),
```

Following #6742 being merged, could you add this line please?

This will ensure that the date and time for posts are localized correctly on all platforms. Thanks!